plansys2_bt_actions/include/plansys2_bt_actions/BTActionNode.hpp

```diff
@@ -134,32 +142,74 @@ class BtActionNode : public BT::ActionNodeBase
   {
     // first step to be done only at the beginning of the Action
     if (status() == BT::NodeStatus::IDLE) {
-      createActionClient(action_name_);
+      // Get the required items from the blackboard
+      unsigned int server_timeout_int;
+      if (!getInput<unsigned int>("server_timeout", server_timeout_int)) {
+        // This will only happen if `providedPorts` is overridden and
+        // the child class does not provide the "server_timeout" port.
+        // Child classes can use the `providedBasicPorts` method to avoid
+        // this issue
+        RCLCPP_INFO(
+          node_->get_logger(),
+          "Missing input port [server_timeout], "
```

Comment:
I would rather prefer to join the messages to drain the 100 characters, instead of occupying more lines. The same in the other RCLCPP_* of this PR

Reply:
What do you mean by "join the messages"?